.github/workflows/ci-formatting-and-dependencies.yml

```diff
@@ -25,16 +25,29 @@ jobs:
           black --check .
           npx prettier --check envision/web/src
 
-  check-requirements-change-linux:
+  update-requirements-linux:
     runs-on: ubuntu-18.04
     if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
     container: huaweinoah/smarts:v0.4.18-minimal
     steps:
+      - name: Install git
+        run: |
+          add-apt-repository -y ppa:git-core/ppa && \
+            apt-get update && \
+            apt-get install -y git
```

Comment:
Why is git being installed?

Reply:
To automatically push changes, we need to git add, git commit, and git push in the ci job. Hence, we need to install git, as our smarts minimal container does not have git pre-installed.